src/script/signingprovider.cpp

```diff
@@ -82,6 +82,18 @@ FlatSigningProvider Merge(const FlatSigningProvider& a, const FlatSigningProvide
     return ret;
 }
 
+void MergeInto(FlatSigningProvider& a, const FlatSigningProvider& b)
+{
+    a.scripts.insert(b.scripts.begin(), b.scripts.end());
+    a.pubkeys.insert(b.pubkeys.begin(), b.pubkeys.end());
+    a.keys.insert(b.keys.begin(), b.keys.end());
+    a.origins.insert(b.origins.begin(), b.origins.end());
+    a.tr_spenddata = a.tr_spenddata;
```

Comment:
```suggestion
    a.tr_spenddata = b.tr_spenddata;
```

Reply:
That’s not correct

Reply:
Yes, I agree. That is really not correct. `b.tr_spenddata` data is merged into` a.tr_spenddata` on the next line.
 But I do not see the point of `a.tr_spenddata = a.tr_spenddata;`.